Dockerfile

```diff
@@ -758,7 +758,7 @@ ARG TARGETARCH
 RUN --mount=type=cache,target=/.cache GOOS=linux GOARCH=${TARGETARCH} go build ${GO_BUILDFLAGS} -ldflags "${GO_LDFLAGS}" -o /installer
 RUN chmod +x /installer
 
-FROM alpine:3.18.4 AS unicode-pf2
+FROM alpine:3.19.1 AS unicode-pf2
```

Comment:
i think this broke xfs, @smira can we update this now?

Reply:
no, let's not update alpine please